templates/base.html

```diff
@@ -1,27 +1,16 @@
 {% load i18n profile_tags static %}
 {% load package_tags %}
+{% load page_metadata_tags %}
 <!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
 <html xmlns="http://www.w3.org/1999/xhtml" dir="{% if LANGUAGE_BIDI %}rtl{% else %}ltr{% endif %}" xml:lang="{{ LANGUAGE_CODE }}" lang="{{ LANGUAGE_CODE }}" {% block angular_header %}ng-app{% endblock angular_header %}>
     <head>
         <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
         <meta name="viewport" content="width=device-width, initial-scale=1">
+        {% block metadata %}{% page_metadata %}{% endblock %}
         {% block angularjs %}
             <script src="{% static 'angular/angular.min.js' %}"></script>
             <script src="{% static 'angular/controllers.js' %}?123"></script>
         {% endblock angularjs %}
-        <title>{% block head_title_base %}{{ SITE_TITLE }} : {% block head_title %}{% endblock %}{% endblock %}</title>
```

Comment:
Why are you removing the title here?

Reply:
I moved it to `templates/includes/page_metadata.html` to reduce redundancy. Otherwise we need to add it like this in every page:

```jinja
{% trans "Search" as trans_title %}

{% block metadata %}
    {% page_metadata page_title=trans_title %}
{% endblock %}

{% block head_title %}{{ trans_title }}{% endblock %}
```

I can revert it back if this is not expected. :)

Reply:
@saadmk11 thanks for the explanation. That makes sense. No need to revert it back, I just wanted to make sure I understood the change! 😁